packages/hawtio/src/ui/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './about'
+export * from './icons'
+export * from './login'
+export * from './notification'
+export * from './page'
```

Comment:
Why are they needed?  Note that it will change the Hawtio API we expose to external plugin authors. So generally it needs to be justified with a clear reason.